packages/nimble-components/src/table-column/duration-text/tests/table-column-duration-text.stories.ts

```diff
@@ -0,0 +1,122 @@
+import { html, ref } from '@microsoft/fast-element';
+import type { Meta, StoryObj } from '@storybook/html';
+import { withActions } from '@storybook/addon-actions/decorator';
+import {
+    createUserSelectedThemeStory,
+    incubatingWarning
+} from '../../../utilities/tests/storybook';
+import { tableTag } from '../../../table';
+import { tableColumnDurationTextTag } from '..';
+import {
+    SharedTableArgs,
+    sharedTableActions,
+    sharedTableArgTypes,
+    sharedTableArgs
+} from '../../base/tests/table-column-stories-utils';
+import { tableColumnTextTag } from '../../text';
+
+const simpleData = [
+    {
+        firstName: 'Homer',
+        lastName: 'Simpson',
+        swearWordCadence: 5400000
+    },
+    {
+        firstName: 'Marge',
+        lastName: 'Simpson',
+        swearWordCadence: 2.59e9
+    },
+    {
+        firstName: 'Bart',
+        lastName: 'Simpson',
+        swearWordCadence: 1800500
+    },
+    {
+        firstName: 'Maggie',
+        lastName: 'Simpson',
+        swearWordCadence: Number.NaN
+    },
+    {
+        firstName: 'Montgomery',
+        lastName: 'Burns',
+        swearWordCadence: 3.78e12
+    }
+];
+
+const metadata: Meta<SharedTableArgs> = {
+    title: 'Incubating/Table Column - Duration Text',
+    decorators: [withActions],
+    tags: ['autodocs'],
+    parameters: {
+        actions: {
+            handles: sharedTableActions
+        }
+    },
+    // prettier-ignore
+    argTypes: {
+        ...sharedTableArgTypes,
+        selectionMode: {
+            table: {
+                disable: true
+            }
+        },
+    },
+    args: {
+        ...sharedTableArgs(simpleData)
+    }
+};
+
+export default metadata;
+
+interface TextColumnTableArgs extends SharedTableArgs {
+    fieldName: string;
+    checkValidity: () => void;
+    validity: () => void;
+}
+
+const durationTextColumnDescription = 'The `nimble-table-column-duration-text` column is used to display duration fields as text in the `nimble-table`. The duration values must be of type `number` and each represent a total number of milliseconds. Values that represent more than 100 days or less than a millisecond will be formatted in scientific notation as seconds only. All sub-second values will be represented with up to 3 digits of precision.';
```

Comment:
If we end up supporting negative values, the language should be updated to say "values with a magnitude of". If we don't support negatives, that should be documented here.

Reply:
No push back on decision to not support negative values.